src/simulation/openables/index.ts

```diff
@@ -38,7 +52,19 @@ const openablesObject = {
 	SinisterChest,
 	BrimstoneChest,
 	LarransChest,
-	SeedPack
+	SeedPack,
+	BabyImpling,
```

Comment:
Wouldn't this be cleaner:
```ts
import { Implings } from './Implings'
...
const openablesObject = {
...
  SeedPack,
  ...Implings
}
```

Reply:
I wanted them as an array, and you cant spread an array over an object like that, not a big deal

Reply:
ah, yep, makes sense